\section{Real data example}
We now apply the proposed methodology to a gene expression dataset containing mRNA and RNAseq expression values for $~4000$ genes, divided into 88 pathways, for 262 ER-positive (ER+) and 76 ER-negative breast cancer patients. (more info on data)

Our objective here is to (a) obtain mRNA-mRNA, mRNA-RNAseq and RNAseq-RNAseq networks for the ER+ and ER- groups while incorporating pathway information, (b) test for differential strengths of mRNA-RNAseq connections between the two sample groups. To this end, we take mRNA and RNAseq expression data as the top and bottm layers (X and Y), respectively, and consider pathway-wise groups. For comparison purposes, we apply JMMLE and the separate estimation method \citet{LinEtal16} for estimating $\cB$ and $\Omega_y$, and JSEM for estimating $\Omega_y$. We use root mean squared errors over 100 random 80:20 train-test splits of samples within each group, as well as percentage of non-zero coefficients in $\widehat \cB$ and $\widehat \Omega_y$ for comparing estimation performances of the methods. To summarize within-layer and between-layer interactions, we take element-wise averages for $\widehat \bfB_k, \widehat \Omega^y_k; k= 1,2$ across the random splits.