\section{Introduction}

Aberrations in complex biological systems develop in the background of diverse genetic and environmental factors and are associated with multiple complex molecular events. These include changes in the genome, transcriptome, proteome and metabolome, as well as epigenetic effects. Advances in high-throughput profiling techniques have enabled a systematic and comprehensive exploration of the genetic and epigenetic basis of various diseases, including cancer \citep{lee2016integrated, kaushik2016inhibition}, diabetes \citep{yuan2014integrated,sas2018shared}, chronic kidney disease \citep{atzler2014integrated}, etc. Further, such multi-Omics collections have become available for patients belonging to different, but related disease subtypes, with The Cancer Genome Atlas (TCGA: \citet{Tomczak15}) being a prototypical one. Hence, there is an increasing need for models that can {\em integrate} such complex data both {\em vertically} across multiple modalities and {\em horizontally} across different disease subtypes.

Figure~\ref{fig:multi2layer} provides a schematic representation of the horizontal and vertical structure of such heterogeneous multi-modal Omics data as outlined above. A simultaneous analysis of all components in this complex layered structure has been coined in the literature as {\it data integration}. While it is common knowledge that this will result in a more comprehensive picture of the regulatory mechanisms behind diseases, phenotypes and biological processes in general, there is a dearth of rigorous methodologies that satisfactorily tackle all challenges that stem from attempts to perform data integration \citep{JoycePalsson06,GomezCabreroEtal14,GligPrzulj15}. A review of the present approaches towards achieving this goal, which are based mostly on specific case studies, can be found in \citet{GligPrzulj15} and \citet{ZhangOuyangZhao17}.

\noindent
\textbf{Contributions}
are two-fold. Firstly, we propose an integrative framework to conduct simultaneous inference for all parameters in {\it multiple} and {\it multi-layer} graphical models, essentially formalizing the structure in Figure~\ref{fig:multi2layer}. We decompose the multi-layer problem into a series of two-layer problems, propose an estimation algorithm for them based on group penalization, and derive theoretical properties of the estimators. Generalizing to group structures on the model parameters allows us to incorporate prior information, \textit{as and when available}, on within-layer or between-layer sub-graph components shared across \textit{some or all} $k=1,\cdots,K$. For biological processes, such information can stem from experimental or mechanistic knowledge (for example a pathway-based grouping of genes). Secondly, we obtain \textit{debiased} versions of within-layer regression coefficients in this two-layer model, and derive their asymptotic distributions using estimates of model parameters that satisfy generic convergence guarantees. Subsequently, we formulate a global test, as well as a simultaneous testing procedure that controls for False Discovery Rate (FDR) to detect important pairwise differences among directed edges between layers.

The novel techniques developed are based on a small number of technical assumptions that are quite general. For example, the model quantities used in our global testing procedure do not necessarily need to be sparse, and instead are only required to have $O(\sqrt{ \text{parameter dimension}/n})$ finite sample error bounds that have become standard in the high-dimensional literature (for example see \citet{LohWainwright12, BasuMichailidis15, BasuEtal19}). The advantage of this %Our proposed framework for knowledge discovery from heterogeneous data sources is highly flexible. The
fiexibility is that components can be switched out to adapt the framework to other technical assumptions. The optional group sparsity assumptions in our estimation technique can be replaced by other structural restrictions (or no restrictions), for example low-rank or low-rank-plus-sparse, as deemed appropriate by the prior dependency assumptions across parameters. As long as these resulting estimates converge to the true parameters at the specified finite-sample rates, they can be used by the developed testing methodology.
%
\begin{figure}
\centering
\includegraphics[]{multi2layer}
\caption{Multiple multilayer graphical models. The matrices $(\bfX^k, \bfY^k, \bfZ^k), k = 1,2,3$ indicate data for each layer and category $k$. Within-layer connections (black lines) are undirected, while between-layer connections (red lines) go from an upper layer to the successive lower layer. For each type of edges (i.e. within $\cX, \cY, \cZ$ and $\cX \rightarrow \cY, \cY \rightarrow \cZ$), there are common edges across some or all $k$. }
\label{fig:multi2layer}
\end{figure}

\noindent
\textbf{Related work:}
Gaussian Graphical Models (GGM) have been extensively used to model biological networks in the last few years. While the initial work on GGMs focused on estimating undirected edges within a single network through obtaining sparse estimates of the inverse covariance matrix from high-dimensional data (e.g. see references in \citet{BuhlmannvandeGeer11}), attention has shifted to estimating parameters from more complex structures. This includes (1) analyzing multiple related but not identical graphical models simultaneously, and (2) stacking up more multiple graphical models to form hierarchical multilayer networks, with both directed and undirected edges. For the first class of problems, \citet{GuoEtal11} and \citet{XieLiuValdar16} assumed perturbations over a common underlying structure to model multiple precision matrices, while \citet{DanaherEtal14} proposed using fused/group lasso type penalties for the same task. To incorporate prior information on the group structures across several graphs, \citet{MaMichailidis15} proposed the Joint Structural Estimation Method (JSEM), which uses group-penalized neighborhood regression and subsequent refitting for estimating precision matrices.
For the second problem, a two-layered structure can be modeled by interpreting directed edges between the two layers as elements of a multitask regression coefficient matrix, while undirected edges inside either layer correspond to the precision matrix of predictors in that layer. While several methods exist in the literature for joint estimation of both sets of parameters \citep{LeeLiu12, CaiEtal12}, only recently \citet{LinEtal16} made the observation that a multi-layer model can, in fact, be decomposed into a series of two-layer problems. Subsequently, they proposed an estimation algorithm and derived theoretical properties of the resulting estimators.

%\paragraph{}
All the above approaches focus either on the horizontal or the vertical dimensions of the full hierarchical structure depicted in Figure~\ref{fig:multi2layer}. Hence, multiple related groups of heterogeneous data sets have to be modeled by analyzing all data in individual layers (i.e. models for $\{ \bfX^k \}$, $\{ \bfY^k \}$, $\{ \bfZ^k \}$), and then separately analyzing individual hierarchies of datasets (i.e. separate models for $(\bfX^k, \bfY^k, \bfZ^k), k = 1,2,3$). In another line of work, \citet{KlingEtal15,ZhangOuyangZhao17} model all undirected edges within all nodes together using penalized log-likelihoods. The advantage of this approach is that it can incorporate feedback loops and connections between nodes in non-adjacent layers. However, it has two considerable caveats. Firstly, it does not distinguish between hierarchies, hence delineating the direction of a connection between two nodes across two
different Omics modalities is not possible in such models. Secondly, computation becomes difficult when data from different Omics modalities are considered, since the number of estimable parameters increases at a faster late compared to a hierarchical model.% (as seen in the numerical examples of \citet{ZhangOuyangZhao17}, which are limited to small feature dimensions ($<$ 70) only {\colrbf is this needed?}).

While there has been some progress for parameter estimation in multilayer models, little is known about the sampling distributions of resulting estimates. Current research on such distributions and related testing procedures for estimates from high-dimensional problems has been limited to single-response regression using lasso \citep{ZhangZhang14,JavanmardMontanari14,JavanmardMontanari18,vanDeGeerEtal14} or group lasso \citep{MitraZhang16} penalties, and partial correlations of single \citep{CaiLiu16} or multiple \citep{BelilovskyEtal16,Liu17} GGMs. From a systemic perspective, testing and identifying downstream interactions that differ across experimental conditions or disease subtypes can offer important insights on the underlying biological process \citep{MaoEtal17,LiEtal15}. In our proposed integrative framework, this can be accomplished by developing a hypothesis testing procedure for entries in the within-layer regression matrices.

%\paragraph{}
%The contributions of this paper are two-fold. Firstly, we propose an integrative framework to conduct simultaneous inference for all parameters in multiple multilayer graphical models, essentially formalizing the structure in Figure~\ref{fig:multi2layer}. We decompose the multi-layer problem into a series of two-layer problems, incorporate prior information on structural similarities through imposing group structures on the model parameters, propose an estimation algorithm for them and derive theoretical properties of the estimators. Secondly, we obtain {\em debiased} versions of within-layer regression coefficients in this two-layer model, and derive their asymptotic distributions using estimates of model parameters that satisfy generic convergence guarantees. Consequently, we formulate a global test, as well as a simultaneous testing procedure that controls for False Discovery Rate (FDR) to detect important pairwise differences among directed edges between layers.
%
%Our proposed framework for knowledge discovery from heterogeneous data sources is highly flexible. Imposing group structures on the model parameters allows us to incorporate prior information on within-layer or between-layer sub-graph components shared across some or all $k$. However, these group sparsity assumptions in our estimation technique can be replaced by other structural restrictions, for example low-rank or low-rank-plus-sparse, as and when deemed appropriate by the prior dependency assumptions across parameters. As long as the resulting estimates converge to corresponding true parameters at certain rates, they can be used by the developed testing methodology.

\noindent
\textbf{Organization of paper:}
We start with the model formulation in Section~\ref{sec:sec2}, then introduce our computational algorithm for a two-layer model, and derive theoretical convergence properties of the algorithm and resulting estimates. In section~\ref{sec:sec3}, we start by introducing the debiased versions of rows of the regression coefficient matrix estimates in our model, then use already computed parameter estimates that satisfy some general consistency conditions to obtain its asymptotic distribution. We then move on to pairwise testing, and use sparse estimates from our algorithm to propose a global test to detect overall differences in rows of the coefficient matrices, as well as a multiple testing procedure to detect elementwise differences and perform within-row thresholding of estimates in presence of moderate misspecification of the group sparsity structure. Sections~\ref{sec:sec4} and \ref{sec:secreal} are devoted to implementation of our methodology. In Section~\ref{sec:sec4}, we evaluate the performance of our estimation and testing procedure through several simulation settings, and give strategies to speed up the computational algorithm for high data dimensions. Section~\ref{sec:sec5} presents a real data example, where we illustrate how the application of our framework leads to knowledge discovery in complex biological networks. We conclude the paper with a discussion in Section~\ref{sec:secreal}. Proofs of all theoretical results, as well as some auxiliary results, are given in the Appendix.

\noindent
\textbf{Notation:}
We denote scalars by small letters, vectors by bold small letters and matrices by bold capital letters. For any matrix $\bfA$, $(\bfA)_{ij}$ denote its element in the $(i,j)\Th$ position. For $a,b \in \BN$, we denote the set of all $a \times b$ real matrices by $\BM(a,b)$. For a positive semi-definite matrix $\bfP$, we denote its smallest and largest eigenvalues by $\Lambda_{\min} (\bfP)$ and $\Lambda_{\max} (\bfP)$, respectively. For any positive integer $c$, define $\cI_c = \{ 1, \ldots, c\}$. For vectors $\bfv$ and matrices $\bfM$, $\| \bfv \|$, $\|\bfv \|_1$ or $\|\bfM \|_1$ and $\|\bfv \|_\infty$ or $\|\bfM \|_\infty$ denote euclidean, $\ell_1$ and $\ell_\infty$ norms, respectively. The notation $\supp(\bfA)$ indicates the non-zero edge set in a matrix (or vector) $\bfA$, i.e. $\supp(\bfA) = \{(i,j): (\bfA)_{ij} \neq 0\}$. For any set $\cS$, $| \cS|$ denotes the number of elements in that set. For positive real numbers $A, B$ we write $A \succsim B$ if there exists $c>0$ independent of model parameters such that $A \geq cB$. We use the `$:=$' notation to define a quantity for the first time.